resources/aws_ec2.go

```diff
@@ -52,6 +52,8 @@ const (
 	SnsPrefix = Ec2Prefix + "sns-"
 	// SnsTopicName is the name of the sns topic created by snsMakeTopic.
 	SnsTopicName = SnsPrefix + "events"
+	// SnsServerShutdownTimeout is the duration in seconds of the timout context for the sns server shutdown.
```

Comment:
timout -> timeout

also, keep in mind you can rephrase user facing comments to real english as opposed to literal technical descriptions. eg: "SnsServerShutdownTimeout is the maximum number of seconds to wait for the http server to shutdown gracefully."